pkgs/top-level/all-packages.nix

```diff
@@ -9494,7 +9502,7 @@ with pkgs;
   libcomps = callPackage ../tools/package-management/libcomps { python = python3; };
 
   libcxxrt = callPackage ../development/libraries/libcxxrt {
-    stdenv = if stdenv.hostPlatform.useLLVM or false
+    stdenv = if stdenv.hostPlatform.cxxlib == "libcxx" && !stdenv.hostPlatform.isDarwin
              then overrideCC stdenv buildPackages.llvmPackages.tools.clangNoLibcxx
              else stdenv;
   };
```

Comment:
Is this really required? It builds fine on Darwin, which uses `libc++`. This condition seems wrong because what if you’re not using Clang? cc @alyssais who introduced this.

Reply:
I think this was to prevent a build error or infinite recursion.